### PR TITLE
Enhancement/import arena

### PR DESCRIPTION
**Pull request by ramirobg94**
[ ] Import button
[ ] import file and create empty survey with name
[ ] test e2e that covers the tasks above

### CODE REVIEW

File: server/modules/arenaImport/service/arenaImport/arenaImportJob.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    })
+  }
+
+  async onEnd() {
+    await super.onEnd()
+
+    const { arenaSurveyFileZip, surveyId } = this.context
+
+    if (arenaSurveyFileZip) {
+      arenaSurveyFileZip.close()
```
Comment (minotogna) [inline server/modules/arenaImport/service/arenaImport/arenaImportJob.js:27]: should this be deleted as well ?

File: server/modules/arenaImport/service/arenaImport/metaImportJobs/surveyCreatorJob.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    const languages = Survey.getLanguages(surveyInfo)
+    const defaultLanguage = Survey.getDefaultLanguage(surveyInfo)
+
+    const labels = Survey.getLabels(surveyInfo)
+    const label = Survey.getLabel(surveyInfo, defaultLanguage)
+
+    const descriptions = Survey.getDescriptions(surveyInfo)
+
+    const newSurveyInfo = Survey.newSurvey({
+      ownerUuid: User.getUuid(this.user),
```
Comment (minotogna) [inline server/modules/arenaImport/service/arenaImport/metaImportJobs/surveyCreatorJob.js:32]: owner remane the one that comes from the zip

**Review by minotogna — CHANGES_REQUESTED** [review]

File: server/modules/arenaImport/service/arenaImport/arenaImportJob.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    await super.onEnd()
+
+    const { arenaSurveyFileZip, surveyId, filePath } = this.context
+
+    if (arenaSurveyFileZip) {
+      arenaSurveyFileZip.close()
+    }
+
+    if (!this.isSucceeded() && surveyId) {
+      await FileUtils.rmdir(filePath)
```
Comment (minotogna) [inline server/modules/arenaImport/service/arenaImport/arenaImportJob.js:32]: shouldn't the file be deleted also if the import is successfully ?

**Review by SteRiccio — APPROVED** [review]

File: server/modules/arenaImport/service/arenaImport/arenaImportJob.js (hunk 3 above)
Comment (ramirobg94) [reply server/modules/arenaImport/service/arenaImport/arenaImportJob.js]: I was thinking that maybe we like to retry with this file but yes! better deleting all the time

**Review by minotogna — APPROVED** [review]

**Pull request merged**